src/ape/__init__.py

```diff
@@ -26,35 +26,34 @@ def __getattr__(name: str):
     if name not in __all__:
         raise AttributeError(name)
 
-    elif name == "reverts":
+    if name == "reverts":
```

Comment:
I've always preferred `elif` in these cases in case you need to switch from returning here to building up a return (have had a rookie mistakes back in the day from that).

Can we disable this rule?